CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 -   Added data type and manual layout tests for the `draw` function.
 -   Updated file paths for example files under docs to import the correct file.
 -   Added style and automatic layout tests for the `draw` function.
+-   Fixed highlight bug and changed implementation of `hide` style option.
```

Comment:
This change a bug fix, so should go under the "Bug fixes" section. You should describe in more detail what the bug is, but don't need to mention specific implementation details. I.e., focus on the _what_, not the _how_.